### DISCUSSION TITLE
Is this project active?

### DISCUSSION THREAD

**Discussion by ghost** [dexie/Dexie.js#1652] [Q&A] [answered]
Hi everyone.

I want to help, but I want to know if this project is active. *Is this project active?*

**Comment by dfahlander** [comment, marked as answer]
Yes, this project is active. For more information, see web page and road map at https://dexie.org and https://dexie.org/roadmap/

**Reply by ghost** [reply]
Hi dfahlander.

> Yes, this project is active. For more information, see web page and road map at https://dexie.org/ and https://dexie.org/roadmap/

thank you for feedback.
